variable.tf

```diff
@@ -1 +1,7 @@
 variable "env_code" {}
+
+variable "vpc_cidr" {}
+
+variable "public_cidr" {}
+
+variable "private_cidr" {}
```

Comment:
could you please, add a newline at the end of file.

Reply:
new line added